src/utils/aotf.js

```diff
@@ -136,6 +137,7 @@ export const mutationIcons = {
   reload: mdiReload,
   remove: mdiMinusCircleOutline,
   resume: mdiPlay,
+  scan: mdiFolderRefresh,
```

Comment:
(We don't actually need this as the scan mutation does not go in the command menu)